### DISCUSSION TITLE
Move newer, untested, or unstable packages to `unstable-repo`

### DISCUSSION THREAD

**Discussion by Yonle** [termux/termux-packages#7794] [General]
A week ago, I've been discuss this issue in gitter and i want to discuss it in github discussion again.

Some new package like `python` version 3.10, `libc++` issue with newer NDK, and some uncommon issue seems to appear in this main package repo. By the way, main package repositories should be tested & confirmed stable, right?

And now we have a major problem with libc++ and i would like to suggest this newer libc++ to get moved to `unstable-repo` in order to reduce crash issue after upgrading.

**Comment by Yonle** [comment]
I think we should revert commit to when newer libc++ is not published, or we may need to do something?

**Comment by ghost** [comment]
You just need to wait until everything will get ready. It is not possible to tell exactly what will be broken if we update something, change compiler, etc. All rebuilds and fixes are performed when the base has been merged.

I'm going to close the "unstable-repo" since it finished doing its purpose long time ago (once we switched to Bintray). It was an intermediate place where all new packages were stored due to https://termux.net ownership issues (in main repo everything was built manually by Fornwall, which introduced a bottleneck). Now that's not an issue anymore. Also as literally no one looks for "fixing" unstable packages, it is better to merge this repo with main.

If you afraid that something is not stable, submit fixes.

> Some new package like python version 3.10

New package != unstable. Welcome to rolling distribution and bleeding edge package versions. You may not upgrade if need specifically old version.

**Reply by Yonle** [reply]
Noted 👀
